pysiaf/scripts/runtest.py

```diff
@@ -0,0 +1,15 @@
+import os
```

Comment:
This file should not be included in the project. If the purpose is to verify the functionality of the function, a dedicated test could be added to the tests/ directory (e.g. `tests/test_match_v2v3.py`)